staging/src/k8s.io/apimachinery/pkg/util/remotecommand/constants.go

```diff
@@ -16,7 +16,11 @@ limitations under the License.
 
 package remotecommand
 
-import "time"
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
 
 const (
 	DefaultStreamCreationTimeout = 30 * time.Second
```

Comment:
@sttts @deads2k are you ok with moving these constants to apimachinery?  @dshulyak seemed to be following @deads2k's suggestion (https://github.com/kubernetes/kubernetes/pull/41331#discussion_r101272665).